source/partials/release_notes/_release-20-7-0.md.erb

```diff
@@ -0,0 +1,49 @@
+<h4>Dashboard Improvements</h4>
+
+Starting GoCD 20.7.0 release, the users will have the ability to trigger manual stages from the dashboard. The page will also provide an overview when clicked on a stage icon for any pipeline.
+
+<figure class='shrink-figure-100'>
+    <%= image_tag 'releases/20.7.0/stage_overview.gif', alt: 'Quick stage overview on dashboard' %>
+</figure>
+
+
+<h4>Other Improvements</h4>
+
+* Starting this release, Alpine 3.12 based docker images for GoCD Agent are <%= link_to 'available', 'https://hub.docker.com/r/gocd/gocd-agent-alpine-3.12' %>.
+* The bundled yum package material plugin has been updated to work on RHEL 8
+* <%= link_to_issue 8411, 'Allow only admins and pipeline group admins to perform check connection on pipeline materials.' %>
+
+<h4>Bug fixes</h4>
+
+* <%= link_to_issue 8416, 'Agent SSL Verification with reverse proxy fails' %>
+* <%= link_to_issue 8427, 'Agents going Lost Contact after a while and warnings in the server about remoteBuildRepository' %>
+* <%= link_to_issue 8460, 'Mask authentication credentials in urls on materials tab on pipeline config edit page' %>
+* <%= link_to_issue 8463, 'Fail to perform any GoCD updates due to invalid config repository configuration' %>
```

Comment:
@kritika-singh3 - there were two parts to this issue:

1. Missing the null check validation for an exec task
2. GoCD stopping cruise-config.xml updates on an invalid config repository.

Should we mention both?

Reply:
Not sure. The point 1 was fixed as part of fixing the issue 8463.